web/views.py

```
@@ -32,7 +32,7 @@
 from django.core.management import call_command
 from django.core.paginator import Paginator
 from django.db import IntegrityError, models, router, transaction
-from django.db.models import Avg, Count, Q, Sum
+from django.db.models import Avg, Case, Count, IntegerField, Q, Sum, When
```

Ruff (0.8.2)

`django.db.models.Case` imported but unused

Remove unused import

(F401)